lib/price-import.coffee

```diff
@@ -70,7 +71,7 @@ class PriceImport extends ProductImport
         fs.outputJsonSync(errorFile, error, {spaces: 2})
 
       if _.isFunction(@errorCallback)
-        @errorCallback(error, @logger)
+        @errorCallback(util.inspect(error, false, null, true), @logger)
```

Comment:
`util.inspect(object, showHidden, depth, colors)`
We don't need the hidden objects, but want to shoe everything nested and I also added some nice coloring...

https://nodejs.org/api/util.html#util_util_inspect_object_options